modules/proc_use-api.adoc

```diff
@@ -101,6 +101,52 @@ Use the following procedure to create an OAuth access token.
 This is the only opportunity to copy and save the access token. It cannot be reobtained after leaving this page. 
 ====
 
+[id="creating-v2-oauth-access-token"]
+== Creating an OCI referrers OAuth access token
+
+In some cases, you might want to create an OCI referrers OAuth access token. This token is used to list OCI referrers of a manifest under a repository.
+
+.Procedure
+
+. Update your `config.yaml` file to include the `FEATURE_REFERRERS_API: true` field. For example:
++
+[source,yaml]
+----
+# ...
+FEATURE_REFERRERS_API: true
+# ...
+----
+
+. Enter the following command to Base64 encode your credentials:
++
+[source,terminal]
+----
+$ echo -n '<username>:<password>' | base64
+----
++
+.Example output
++
+[source,terminal]
+----
+abcdeWFkbWluOjE5ODlraWROZXQxIQ==
+----
+
+. Enter the following command to use the base64 encoded string and modify the URL endpoint to your {productname} server:
++
+[source,terminal]
+----
+$ curl --location '<quay-server.example.com>/v2/auth?service=<quay-server.example.com>&scope=repository:quay/listocireferrs:pull,push' --header 'Authorization: Basic cXVheWFkbWluOjE5ODlraWROZXQxIQ==' -k | jq
+----
```

Comment:
Suggest clarify that the authorization should be the base64 encode of "username:password"

Reply:
Done ty